lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  --policy-indicator-highlight-color: #{$nx-threat-color-unspecified};
+  align-items: center;
+  border: 1px solid var(--policy-indicator-highlight-color);
+  border-radius: $nx-border-radius;
+  display: inline-flex;
+  height: 30px;
+  margin: 0 0 $nx-spacing-xs $nx-spacing-md;
+  overflow: hidden;
+  width: max-content;
+  max-width: 100%;
+}
+
+.nx-policy-violation-indicator__badge {
+  align-self: stretch;
+  align-items: center;
+  background-color: var(--policy-indicator-highlight-color);
+  display: inline-flex;
+
+  .nx-icon {
+    color: #fff;
+  }
+}
+
+.nx-policy-violation-indicator__text {
+  @include semi-bold();
+  @include nx-truncate-ellipsis;
+
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-md;
+  text-transform: uppercase;
+  // max width of element from design is 859px, subtract the badge, the padding, and the borders.
+  max-width: 793px;
```

Comment:
Is this still needed?

Reply:
> Is this still needed?

Yes because the max-width for the component as specified by Zeplin is less than the maximum width of some of the element's potential containers.

Reply:
Maybe I wasn't clear, but part of what I was trying to say in the other thread on this line (https://github.com/sonatype/sonatype-react-shared-components/pull/229/files#r533511093) was that I don't think the max as specified in zeplin is valuable to have.   The important thing is that the indicator doesn't overflow it's container, which we now handle via `max-width: 100%`.  Beyond that, I'm not sure it's helpful to limit the displayable size of its contents.

Reply:
> Maybe I wasn't clear, but part of what I was trying to say in the other thread on this line (https://github.com/sonatype/sonatype-react-shared-components/pull/229/files#r533511093) was that I don't think the max as specified in zeplin is valuable to have. The important thing is that the indicator doesn't overflow it's container, which we now handle via `max-width: 100%`. Beyond that, I'm not sure it's helpful to limit the displayable size of its contents.

IMO this is a question for @dturner32 and he needs to be tagged or he won't see it.

Reply:
> > Maybe I wasn't clear, but part of what I was trying to say in the other thread on this line (https://github.com/sonatype/sonatype-react-shared-components/pull/229/files#r533511093) was that I don't think the max as specified in zeplin is valuable to have. The important thing is that the indicator doesn't overflow it's container, which we now handle via `max-width: 100%`. Beyond that, I'm not sure it's helpful to limit the displayable size of its contents.
> 
> IMO this is a question for @dturner32 and he needs to be tagged or he won't see it.

I'd argue for keeping the max width setting to keep the content in the indicator from stretching all the way across the page.